pandas/core/frame.py

```diff
@@ -4673,6 +4673,49 @@ def drop_duplicates(
         See Also
         --------
         DataFrame.value_counts: Count unique combinations of columns.
+
+        Examples
+        --------
+
+        Consider dataset containing ramen rating.
+
+        >>> brand = ['Yum Yum', 'Yum Yum', 'Indomie', 'Indomie', 'Indomie']
+        >>> df = pd.DataFrame({'brand': brand,
+        ... 'style': ['cup', 'cup', 'cup', 'pack', 'pack'],
+        ... 'rating': [4, 4, 3.5, 15, 5]},
+        ...  index=['TH', 'TH', 'ID', 'ID', 'ID'])
+
```

Comment:
Remove the empty line, so it won't split into sections

Reply:
I missed that, I'm gonna fix this issue.